feature_engine/transformation/ciclycal.py

```diff
@@ -0,0 +1,122 @@
+from typing import List, Optional, Union
```

Comment:
`cyclical.py` instead of ciclycal.py as file name?

Reply:
OK!